### ISSUE TITLE
App crashes when not running for XCode

### ISSUE THREAD

**Issue by hirenhcm** [patchthecode/JTAppleCalendar#368]
Hi,

I am getting a small issue when using JTAppleCalendar and need your help. I have followed the tutorial that is provided in the documentation. 

I am able to run the app on my simulator and see the basic calendar. The app only runs when I run it from XCode in debug mode. But when I try to run the same app on the simulator, without XCode, it crashes.

I am not able to provide any logs as it is crashing only when not connected to XCode. The app just launches and stops.

I have installed the lib without using cocopods.  Just dragged the xcodeproj file in my project. 

I am using XCode Version 8.3 and running it on iPhone 7 Plus simulator on iOS  10.3.

**Comment by patchthecode** [comment]
If its a small project then mind zipping it and sending it over?

**Comment by hirenhcm** [comment]
Thanks for replying. 
Yes it is just a tutorial. PFA.
[CalenderDemo.zip](https://github.com/patchthecode/JTAppleCalendar/files/888573/CalenderDemo.zip)

**Comment by patchthecode** [comment]
While there are many different ways to setup this manually, the way you have done it is not the way that I am used to.

How i usually do it manually is like this:
This project has a `Sources` folder.
1. simply drag the sources folder into your project.
2. You will likely get a lot of errors at this point; therefore go to every file in the sources folder and put this at the top 'Import UIKit`. I think thats the only import needed to silence the errors.

Done.

**Comment by hirenhcm** [comment]
@patchthecode Now the App works without XCode in debug mode when I set up the project the way you mentioned it.

But I am curious on why it fails if imported using xcodeproj. As I have used other lib by adding xcodeproj file to my project.

For now, it works and I can get to my development. Thanks for your help. If I find the root cause of the above issue I will share it here.

But for now, thank you until I have any other questions. 

Respect.
